modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -274,7 +280,7 @@
 			to_chat(user, span_warning("You cannot do multiple things at the same time!"))
 			return
 		in_use = TRUE
-		if(sinew_lower_chance >= 100) //max is 100
+		if(sinew_lower_chance >= (MAX_UPGRADE_SINEW * 10)) //max is 100
```

Comment:
Same comment as previous.

Reply:
Actually, I think the biggest thing is that sinew_lower_chance feels like a weird name for that variable.

Reply:
sinew_lower_chance could be changed to sinew_temp_chance I guess, but its slowly becoming more not atomized with these changes. I definitely agree it could be named better, but that should be for an entirely different PR.